installation.rst

```diff
@@ -67,8 +95,12 @@ Testing installation
 --------------------
 
 It is strongly advised to run the vispy test suite right after installation to
-check if everything is ok. To do this, just type::
+check if everything is ok. To do this, just type:
+
+.. code-block:: python
```

Comment:
FYI this one should be unnecessary because the default for the repo should be Python anyway. But no harm in having it

Reply:
Yeah the code at the top uses the code block directive for python so was trying to be consistent.

Reply:
I guess it makes the GitHub editor render it nicely, too:

https://github.com/davidh-ssec/vispy-website/blob/05614fa928b7351df32ea7df21eda5870b6a4372/installation.rst